amundsen_application/static/css/_inputs.scss

```diff
@@ -1,5 +1,7 @@
 @import 'variables';
 
+$testVAR: 0;
```

Comment:
What's this?

Reply:
Oops! 
Part of a test that I forgot to clean up!

Thanks!